docs/roles/project-coordinator.rst

```diff
@@ -8,32 +7,43 @@ project lead for a specific project. This page explains more about the
 responsibilities and role of the project coordinator.
 
 
-Description
+Description 
 -----------
 
-The project coordinator is responsible for ensuring that all club projects have the resources needed to continue developing. This may include monetary resources, connections to mentors or the like and infrastructure to host the projects. 
+The project coordinator is responsible for ensuring that all club projects have
+the resources needed to continue developing.  This may include monetary
+resources, connections to mentors or the like and infrastructure to host the
+projects. 
 
 Eligibility requirements
 ^^^^^^^^^^^^^^^^^^^^^^^^
 
-The project coordinator must maintian good academic standing as defined by RIT to remain a member of eboard.
-Due to the hands on nature of the role, the project coordinator must be available in person during club and eboard meetings. If needed, a temporary replacement may be appointed to fill the role while the project coordinator is out on coop. In that case there should be good communication between the the two people in order to facilitate a smooth transition.
+The project coordinator must maintain good academic standing as defined by RIT
+to remain a member of eboard.  Due to the hands on nature of the role, the
+project coordinator must be available in person during club and eboard
+meetings.  If needed, a temporary replacement may be appointed to fill the role
+while the project coordinator is out on coop. In that case there should be good
+communication between the the two people in order to facilitate a smooth 
+transition.
 
 
-Responsibilities
+Responsibilities 
 ----------------
 
-* Be physically present at all eboard meetings
-* Be present at all RITLUG meetings unless a conflict has come up and eboard has been notified
-* Be available to RITLUG members
-* Act as liason between project leads and the rest of eboard
-* Assist with general eboard tasks as needed
+- Act as liason between project leads and the rest of eboard 
+- Mediate disputes between project leads and team members
+- Encourage members to start and participate in club projects
+- Assist project leads in managing their projects
+- Assist project leads in establishing project specific meetings
+- Provide general support to project leads and participants
 
-Resources
+Resources 
 ---------
 
-< any resources, pages, or other things someone coming into this should read to
-be informed and prepared to take on the role – probably best as a list of
-links! >
+`The runbook`_. .._`The runbook`: https://github.com/RITlug/runbook/
+A sample `RITlug project`_. .._`RITlug project`: https://github.com/RITlug/TigerOS
+`Freenode webchat`_. if you get stuck on any project specific things .._`Freenode webchat`: https://webchat.freenode.net/
+The `TigerOS landing page`_. .._`TigerOS landing page`: https://tigeros.ritlug.com
+#fedora-respins on freenode was helpful for questions with building TigerOS
```

Comment:
I might have given confusing instructions on the hyperlinks. In the main part of the section, you insert the hyperlink. Then, at the bottom, you add the comments with the actual URI for the resource. As it is now, these won't render correctly.

Example:

```rst
- `Runbook`_: Details on how to run RITlug
- Something else: Helpful thing
- Another `thing`_: This tells you everything to know about everything

.. _`Runbook`: http://runbook.ritlug.com/
.. _`thing`: https://www.google.com/
```